de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/Order.java

```diff
@@ -0,0 +1,9 @@
+package de.dashup.shared.DatabaseModels;
+
+public interface Order {
```

Comment:
Obsolete, since we have decided to delete the predecessor. 
Besides an order describes only how elements are arranged and does not have any of these attributes. And the attribute `id` is already implemented in the `DatabaseObject` interface.

Reply:
This was just to define an interface for layout objects to be managed by my layout algorithm. So I actually used it for arranging elements. But yes you are right, this is obsolete now.